frontend/src/Routes/KnowledgeCenterRoutes.tsx

```diff
@@ -0,0 +1,145 @@
+import { DeclareAuthenticatedRoutes } from './Routes';
+import AdminLayer1 from '@/Pages/AdminLayer1';
+import {
+    getAdminLevel1Data,
+    getLibraryLayoutData,
+    getStudentLevel1Data
+} from '@/routeLoaders';
+import LibraryLayout from '../Components/LibraryLayout';
+import HelpfulLinks from '../Pages/HelpfulLinks';
+import OpenContentManagement from '@/Pages/OpenContentManagement';
+import Error from '@/Pages/Error';
+import VideoManagement from '@/Pages/VideoManagement';
+import HelpfulLinksManagement from '@/Pages/HelpfulLinksManagement';
+import { FeatureAccess } from '@/common';
+import { AdminRoles, AllRoles } from '@/useAuth';
+import FavoritesPage from '@/Pages/Favorites';
+import LibraryViewer from '@/Pages/LibraryViewer';
+import VideoViewer from '@/Components/VideoEmbedViewer';
+import VideoContent from '@/Components/VideoContent';
+import OpenContent from '@/Pages/OpenContent';
+import StudentLayer1 from '@/Pages/StudentLayer1';
+
+export const KnowledgeCenterAdminRoutes: RouteObject =
+    DeclareAuthenticatedRoutes(
+        [
+            {
+                path: 'knowledge-insights',
+                element: <AdminLayer1 />,
+                loader: getAdminLevel1Data,
+                handle: {
+                    title: 'Knowledge Insights'
+                }
+            },
+            {
+                path: 'knowledge-center-management',
+                element: <OpenContentManagement />,
+                handle: {
+                    title: 'Knowledge Center Management'
+                },
+                children: [
+                    {
+                        path: 'libraries',
+                        loader: getLibraryLayoutData,
+                        element: <LibraryLayout />,
+                        errorElement: <Error />,
+                        handle: {
+                            title: 'Libraries Management'
+                        }
+                    },
+                    {
+                        path: 'videos',
+                        loader: getLibraryLayoutData,
+                        element: <VideoManagement />,
+                        handle: {
+                            title: 'Videos Management'
+                        }
+                    },
+                    {
+                        path: 'helpful-links',
+                        element: <HelpfulLinksManagement />,
+                        handle: {
+                            title: 'Helpful Links Management'
+                        }
+                    }
+                ]
+            }
+        ],
+        AdminRoles,
+        [FeatureAccess.OpenContentAccess]
+    );
+
+export const KnowledgeCenterRoutes: RouteObject = DeclareAuthenticatedRoutes(
+    [
+        {
+            path: 'trending-content',
+            element: <StudentLayer1 />,
+            loader: getStudentLevel1Data,
+            handle: {
+                title: 'Trending Content'
+            }
+        },
```

Comment:
@PThorpe92 Given that we're going to make a short-term change on another PR to create a "Home" page that displays data from Knowledge Center trends..how would this change once that is in place?

Making sure @CK-7vn sees this too.